Sushi.Mediakiwi.Data.Elastic/Repositories/NotificationRepository.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Nest;
+using Sushi.Mediakiwi.Data.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sushi.Mediakiwi.Data.Elastic.Repositories
+{
+    /// <summary>
+    /// Provides methods to store and retrieve instances of <see cref="Notification"/> from an Elastic server. 
+    /// It is advised to use one instance of <see cref="NotificationRepository"/> per application or at least per Elastic connection.
+    /// </summary>
+    public class NotificationRepository : INotificationRepository
+    {
+        private Nest.IElasticClient _client;
+        private string _dataStreamName;
```

Comment:
should be readonly, since it's only set from the CTor

Reply:
agree